MekHQ/src/mekhq/campaign/personnel/Bloodname.java

```diff
@@ -524,13 +524,18 @@ public Clan() {
 		nameChanges = new ArrayList<DatedRecord>();
 	}
 
+    @Override
+    public int hashCode() {
+        return code.hashCode();
+    }
+
```

Comment:
This is not related to images or the other changes – it's just that I noticed this class overrides `equals()` but not `hashCode()`